packages/volto/locales/ca/LC_MESSAGES/volto.po

```diff
@@ -3979,16 +3979,16 @@ msgstr "Pengeu una imatge principal al camp de contingut 'Imatge principal'."
 msgid "Upload files"
 msgstr "Carregueu fitxers"
 
-#. Default: "Uploading files"
-#: components/manage/Contents/ContentsUploadModal
-msgid "Uploading files"
-msgstr "Carregant fitxers"
-
 #. Default: "Uploading image"
 #: components/manage/Blocks/Image/Edit
 msgid "Uploading image"
 msgstr "S'està carregant la imatge"
 
+#. Default: "Uploading {uploadedFiles} Files Out of {totalFiles}"
```

Comment:
For all instances of this phrase throughout the pull request, not just this one suggestion and file, please use sentence casing.

```suggestion
#. Default: "Uploading {uploadedFiles} files out of {totalFiles}"
```

Reply:
okay. I'll change this up.